.golangci.yml

```diff
@@ -51,7 +51,11 @@ issues:
     - path: pkg/util/containerd
       text: "name will be used as .* by other packages, and that stutters"
     # TODO: Needs manual update
-    - text: "package github.com/golang/protobuf/proto is deprecated: .*"
+    - text: "\"github.com/golang/protobuf/proto\" is deprecated"
+      linters: [staticcheck]
+    - text: "rand.Seed has been deprecated since Go 1.20"
+      linters: [staticcheck]
+    - text: "Temporary has been deprecated since Go 1.18"
       linters: [staticcheck]
```

Comment:
Could you please add a comment to explain why we're escaping it and not fixing it please ?